src/components/chats/components/CreateNewChat/CreateChat.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC } from 'react'
+
+import { Button } from '@chakra-ui/react'
+import { DocumentIcon } from '@heroicons/react/24/outline'
+import { useCreatePrivateChat } from '../../chatHooks'
+import { NewChatProps } from './types'
+import { useCanCreateChat } from './useCanCreateChat'
+
+export const CreateChat: FC<NewChatProps> = ({
```

Comment:
should we name it `CreateChatbotChat`, to make the difference more explicit

Reply:
mm, not sure if that's more specific, probably we should find other names (PrivateChat, SharedChat) for example

Reply:
Yeah, both "SharedChat" or "ChatbotChat" work for me. I would care less about the exact name, more about creating the vocabulary for the domain and applying it consistently. Happy with whatever you choose.